docs/data/base/components/textarea-autosize/UnstyledTextareaIntroduction.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { TextareaAutosize as TextareaAutosizeUnstyled } from '@mui/base/TextareaAutosize';
+import { styled } from '@mui/system';
+
+const blue = {
+  100: '#DAECFF',
+  200: '#b6daff',
+  400: '#3399FF',
+  500: '#007FFF',
+  600: '#0072E5',
+  900: '#003A75',
+};
+
+const grey = {
+  50: '#f6f8fa',
+  100: '#eaeef2',
+  200: '#d0d7de',
+  300: '#afb8c1',
+  400: '#8c959f',
+  500: '#6e7781',
+  600: '#57606a',
+  700: '#424a53',
+  800: '#32383f',
+  900: '#24292f',
+};
+
+const TextareaAutosize = styled(TextareaAutosizeUnstyled)(
```

Comment:
I think we could also move the styles to the bottom of the file ~ all the demos were updated this way recently: https://github.com/mui/material-ui/issues/36498

Reply:
Ah, yes, great point, let's do this, it makes a lot of sense to me.